python/rebCurrentLimits.py

```diff
@@ -36,20 +36,20 @@ def __init__(self, rebps, ts8):
         self.rebps = rebps
         self.ts8 = ts8
         self.logger = rebps.logger
-        self['DigI'] = ChannelLimits('digital.IaftLDO', 430., 650., 100.)
-        self['AnaI'] = ChannelLimits('analog.IaftLDO', 530., 615., 50.)
-        self['ClkHI'] = ChannelLimits('clockhi.IaftLDO', 80., 92., 25.)
-        self['ClkLI'] = ChannelLimits('clocklo.IaftLDO', 32., 50., 25.)
-        self['ODI'] = ChannelLimits('OD.IaftLDO', 7., 13., 10.)
-        self['HtrI'] = ChannelLimits('heater.IaftLDO', 0., 15., 0.)
-#        #- new values and change to IbefLDO
-#        self['DigI'] = ChannelLimits('digital.IbefLDO', 450., 560., 100.)
-#        self['AnaI'] = ChannelLimits('analog.IbefLDO', 500., 660., 50.)
-#        #- setting ClkHI anomolously high for REB4 board on aliveness bench
-#        self['ClkHI'] = ChannelLimits('clockhi.IbefLDO', 90., 180., 25.)
-#        self['ClkLI'] = ChannelLimits('clocklo.IbefLDO', 35., 55., 25.)
-#        self['ODI'] = ChannelLimits('OD.IbefLDO', 7., 15.5, 10.)
-#        self['HtrI'] = ChannelLimits('heater.IbefLDO', 0., 15., 0.)
+#        self['DigI'] = ChannelLimits('digital.IaftLDO', 430., 650., 100.)
```

Comment:
Is it ok to delete these commented out lines or do we want them here for historical purposes?

Reply:
okay to delete